src/sso/sso.spec.ts

```diff
@@ -116,5 +116,45 @@ describe('SSO', () => {
         });
       });
     });
+
+    describe('promoteDraftConnection', () => {
+      it('send valid token', async () => {
+        const mock = new MockAdapter(axios);
+        mock.onPost('/draft_connections/convert').reply(201);
+
+        const token = 'wOrkOStoKeN';
+        const workos = new WorkOS('sk_test_Sz3IQjepeSWaI4cMS4ms4sMuU');
+        await workos.sso.promoteDraftConnection({ token });
+
+        expect(mock.history.post.length).toBe(1);
+        const post = mock.history.post[0];
+        const res = JSON.parse(post.data);
+        expect(res.id).toEqual(token);
+      });
+
+      it('send bad token', async () => {
+        const mock = new MockAdapter(axios);
+        mock
+          .onPost('/draft_connections/convert')
+          .reply(
+            400,
```

Comment:
Maybe better to make this a 404 since we throw that on the other side for a bad id.